### ISSUE TITLE
[Enhancement]: custom header name

### ISSUE THREAD

**Issue by burghy86** [advplyr/audiobookshelf#1102]
### Describe the feature/enhancement

is possible change name audiobookshelf with a custom name of my library

https://i.imgur.com/aqIlZbP.png

with:

https://i.imgur.com/N0hSu7I.png

naturally in the footer add power by audiobook shelf

**Comment by burghy86** [comment]
would anyone like this implementation?

**Comment by chandz05** [comment]
This would be great! Especially now since I also host my eBook library on the app

**Comment by burghy86** [comment]
This is unlikely to be added especially the favicon and logo but you can add that to https://github.com/advplyr/audiobookshelf/issues/1102 it'll stay open

@advplyr can i ask why?

**Comment by holzerseb** [comment]
Would love some customizations for name/logo/custom css too!

**Comment by pwinnski** [comment]
I think "audiobookshelf" is as much a part of the branding as the logo, personally, but cosmetic changes like this are exactly why Violentmonkey and similar tools exist.

**Comment by MikeGold** [comment]
I'd like my custom name as well. Don't mind "powered by audio bookshelf" or whatever, but would be nice if I could name my library whatever I want.

**Comment by c-schaer** [comment]
+1. It's a small feature in the long run, but it's nice to have a "brand" like I have with nextcloud and jellyfin for the family to use

**Comment by imagicalFantasy** [comment]
+1 Could be nice !

**Comment by MikeGold** [comment]
> +1 Could be nice !

Forget it. I'm using Audiobookshelf for years now, and although it's really good, it didn't ever introduced a single new valuable feature.

**Comment by GammaGriffin** [comment]
I too would like to have the ability to do that

**Comment by MikeGold** [comment]
Hey guys, just to let you know, some creative guy made the custom theme for audiobookshelf. It made it look much better, allows for custom logo and library name as well. You can find it on audiobookshelf reddit ;]

**Comment by pwinnski** [comment]
I assume you mean NanoHive, which is not exactly a "theme" so much as an always-running proxy that requires updating whatever means one uses to access ABS remotely. 

Personally, I'll wait for the complete UI React rewrite to be finished, which will, of course, completely break this proxy when released.

**Comment by MikeGold** [comment]
> I assume you mean NanoHive, which is not exactly a "theme" so much as an always-running proxy that requires updating whatever means one uses to access ABS remotely.
> 
> Personally, I'll wait for the complete UI React rewrite to be finished, which will, of course, completely break this proxy when released.

Well, NanoHive does everything that Audiobookshelf was missing for me for the last few years. To be honest I haven't seen a single update in Audiobookshelf since I started using it three years ago that would change anything at all for me and my experience. This guy managed to apply book ratings and reviews within 24 hours from when I requested it. A feature me and many others requested for years. When ABS will update he will upgrade it as well, and if he won't I'll consider freezing my ABS at compatible version.

**Comment by nartb** [comment]
+1 I would really appreciate this feature as it's something I do with my other self-hosted apps and helps me organize things for my friends and family

**Comment by burghy86** [comment]
Hello. I opened this feature request back in 2022. It has consistently been ignored by @advplyrP, and other issues have been closed as "not planned." How can we establish at least some constructive dialogue with @advplyrP to understand the outright rejection of such a simple modification to their fantastic project? Are they afraid of the project being stolen or resold? It would be interesting to have a discussion, if they were open to it.

**Comment by nichwall** [comment]
I can't speak for advplyr, but you can always patch in support in support for yourself by running or building the web client from source. This would only affects the web client and would not change anything for mobile apps due to apps using static icons at build time.

There are alternative web clients that people have created and posted to GitHub and Reddit as well that just interact with the backend server through the API (same as the web client that ships with the server). Not sure if any of them support changing the icon but something you could check out. I don't have any off the top of my head but they've been getting a lot more common lately with the rise of LLM coding.

The simple answer, but probably not what you want to hear, is this is marked as backlog and there are currently over 1500 open issues and PRs between the server and app repositories. At the end of the day, this is an open source hobby project and it would be another feature to maintain. This seems like it would be a low maintenance, but to me it also seems like an edge case based on the amount of engagement on this issue (especially compared to other issues and feature requests) when there are 3rd party options and you can always just patch it in yourself.

**Comment by MikeGold** [comment]
> I can't speak for advplyr, but you can always patch in support in support for yourself by running or building the web client from source. This would only affects the web client and would not change anything for mobile apps due to apps using static icons at build time.
> 
> There are alternative web clients that people have created and posted to GitHub and Reddit as well that just interact with the backend server through the API (same as the web client that ships with the server). Not sure if any of them support changing the icon but something you could check out. I don't have any off the top of my head but they've been getting a lot more common lately with the rise of LLM coding.
> 
> The simple answer, but probably not what you want to hear, is this is marked as backlog and there are currently over 1500 open issues and PRs between the server and app repositories. At the end of the day, this is an open source hobby project and it would be another feature to maintain. This seems like it would be a low maintenance, but to me it also seems like an edge case based on the amount of engagement on this issue (especially compared to other issues and feature requests) when there are 3rd party options and you can always just patch it in yourself.

You are absolutely right, but you have to understand that for 99% normal users there was no update that would bring any value to audiobookshelf for many years now, hence the general frustration. For me personally Audiobookshelf simply worked for many years, but was missing many crucial features like customization or ratings. Some time ago one guy in a very short time built NanoHive, a selfhosted service that takes over calls made to audiobookshelf and use them to totally rebuild Audiobookshelf website. Surprisingly this one guy in a short time managed to incorporate all the features users begged for in here for years. Basically because of that I stopped looking into Audiobookshelf changelogs. Simply it's like talking to a wall. Anything that would bring a proper value to the project is totally ignored, while when I asked the author of NanoHive to incorporate book ratings and other things, he literally added everything in less than a week. It would be nice to have NanoHive run from the same container as Audiobookshelf, but I guess that's asking for too much.

**Comment by nichwall** [comment]
> Some time ago one guy in a very short time built NanoHive, a selfhosted service that takes over calls made to audiobookshelf and use them to totally rebuild Audiobookshelf website.

Yes, that is why there is an API (and a lot of 3rd party apps). If someone wants to build a custom client that interfaces with the bacend, they are welcome to.
